src/components/Utilities/Currency.jsx

```diff
@@ -11,16 +11,26 @@ export const Currency = ({
     locale = userLocale,
     shouldRemoveTrailingZeroes = true,
     maximumFractionDigits = 2,
+    shorten = false,
     children,
 }) => {
     let amount = children;
     if (almostZero(amount)) {
         amount = 0;
     }
 
-    let formattedAmount = numberFormat(amount, currency, locale, isZeroDecimal(currency) ? 0 : maximumFractionDigits);
-    formattedAmount = shouldRemoveTrailingZeroes ? formattedAmount.replace(".00", "") : formattedAmount;
+    const maxDigits = isZeroDecimal(currency) ? 0 : maximumFractionDigits;
+    let formattedAmount = numberFormat(amount, currency, locale, maxDigits, shorten);
+    if (shorten) {
+        return (
+            <span className="ui-currency">
+                {getSymbol(currency, locale)}
+                {formattedAmount}
+            </span>
+        );
+    }
 
+    formattedAmount = shouldRemoveTrailingZeroes ? formattedAmount.replace(".00", "") : formattedAmount;
```

Comment:
Because when we have `maximumFractionDigits=2` we still want to show `$100.00` as `$100`. This param has been there for a while.

```
Intl.NumberFormat("en-US", { maximumFractionDigits: 2, style: 'currency', currency: 'USD' }).format(100); // Returns $100.00 and we want $100
```

Reply:
Oh, then we need `minimumFractionDigits: 0` in that case 😄 

```ts
Intl.NumberFormat("en-US", { minimumFractionDigits: 0, maximumFractionDigits: 2, style: 'currency', currency: 'USD' }).format(100);
```